Adding Everything Presence One and Adonno Tagreader

mscottco opened the pull request:
Adding the above devices based on the revised spec in #392. The articles are a little sparse but needed so the GraphQL database will pickup the new tags for the revised device info page.

Aware the Everything Presence One isn't officially "Made for Esphome" (as far as I know) but needed for UI testing...

Should have a new device info page ready to submit - just wanted to test with these new items first...

tekmaven commented:
I think we need more content on the pages before merging them in.  Could you just use these to test locally or update the frontmatter of existing devices?

mscottco commented:
I've tested locally with these changes on the new UI and it all works okay... but I did find a case where the site breaks if you try and query a tag that doesn't exist in the schema. As such, I was hoping I could stage a couple of devices so the schema is updated before submitting the revised device-page code.

Is there anything specifically you were looking for to be added?

jesserockz commented on src/docs/devices/Adonno-Tagreader/index.md:
```diff
@@ -0,0 +1,12 @@
+standard: global
+board: esp8266
+difficulty: 1
+made-for-esphome: false
```
This is actually true

mscottco replied:
Oh sorry, I couldn't see anywhere it was officially stated on the page but I'll correct it.

jesserockz commented on src/docs/devices/Everything-Presence-One/index.md:
```diff
@@ -0,0 +1,19 @@
+standard: global
+board: esp32
+difficulty: 1
+made-for-esphome: true
```
This is correct

tekmaven commented:
Thanks for the review @jesserockz!

@mscottco I was just thinking the pages would have some ESPHome yaml. I believe all other device pages on the site contain yaml.  I can help add the EP1 yaml when I get some free time if you would like a hand on that.

mscottco commented:
Yeah that's a fair point... in the case of made-for-esphome devices, would it make sense to link to a dashboard-import url so we're not duplicating code? In case the dev changes the main code in the shipping product

tekmaven commented:
I think your point is fair though and at least linking to the dashboard import url will give at least a link to yaml.

mscottco commented:
Fair point. I've updated both devices to include links to the projects yaml.

Thinking about it more, I've changed the project-url at the top to be the board yaml for both projects. I would propose that as a general rule, the project-url should point to the yaml code for the device or to a page with a clear link to the yaml code (ie. a GitHub repo) for better discoverability. Shop urls can go in the main code body if there's an option to purchase a pre-built version.

@jesserockz and @tekmaven, would you agree with this? If so, I'll add something along those lines to the adding-devices.md as part of the main code push.

tekmaven commented:
This looks good to me.  Thanks!
